app/controllers/users.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../logger');
+const { createUser } = require('../services/users');
+const { validatePassword, validateEmail, existEmail } = require('../validations/users');
+
+exports.createUser = async (req, res, next) => {
+  try {
+    const { name, lastName, email, password } = req.body;
+    logger.info(`Init create user by email: ${email}`);
+    validateEmail(email);
+    validatePassword(password);
+    await existEmail(email);
+    const user = await createUser(name, lastName, email, password);
```

Comment:
This could be passed as a single parameter, as an object `{ name, lastName, email, password }`, so you also receive it as an object in the service and you pass it directly to the user creation function.